src/api/generated/apis/daemons-api.ts

```diff
@@ -57,16 +56,15 @@ export class DaemonsApiService extends ApiBaseService {
      * 
      * @summary Restart a background process
      * @param {number} daemonId Numeric ID of the daemon to restart
-     * @param {ExtraBodyParams} [extraBodyParams] A JSON object containing list of additional parameters
      */
-    public async restartDaemon(daemonId: number, extraBodyParams?: ExtraBodyParams): Promise<ApiResponse<void>> {
+    public async restartDaemon(daemonId: number): Promise<ApiResponse<void>> {
         if (daemonId === null || daemonId === undefined) {
             throw new ArgumentNullException('daemonId', 'restartDaemon');
         }
         const queryString = [].join('&');
         const requestUrl = '/daemons/{daemon_id}/restart' + (queryString? `?${queryString}` : '');
 
-        const response = await this.post <void, ExtraBodyParams>(requestUrl.replace(`{${"daemon_id"}}`, encodeURIComponent(String(daemonId))), extraBodyParams);
+        const response = await this.post <void>(requestUrl.replace(`{${"daemon_id"}}`, encodeURIComponent(String(daemonId))));
```

Comment:
Are you sure this `post` call without a second parameter is working?
Have you tested the consumptiom of this SDK version from a client app?
It seems to me it shouldn't work.

Reply:
> Are you sure this `post` call without a second parameter is working?
> Have you tested the consumptiom of this SDK version from a client app?
> It seems to me it shouldn't work.

Yes, in the file `/src/services/ApiBaseService.ts` I updated the `post` method to set the second parameter type to `undefined` by default. In my tests it's works.

Reply:
Oops, my bad then. LGTM!